### PR TITLE
Added support for getting multiple values in a single call

### PR DESCRIPTION
**Pull request by ipostelnik**

### CODE REVIEW

File: src/ascii.rs [hunk 1]
```diff
@@ -60,6 +61,50 @@ where
         Ok(buffer)
     }
 
+    /// Returns values for multiple keys in a single call as a `HashMap` from keys to found values.
+    /// If a key is not present in memcached it will be absent from returned map.
+    pub async fn get_multi<'a, K: Display>(&'a mut self, keys: Vec<&'a K>) -> Result<HashMap<String, Vec<u8>>, Error> {
+        // Send command
+        self.io.write_all("GET".as_bytes()).await?;
+        for k in keys {
+            self.io.write(&[32; 1]).await?;
```
Comment (vavrusa) [inline src/ascii.rs]: `b" "` (I guess this is supposed to be space)

File: src/ascii.rs [hunk 2]
```diff
@@ -60,6 +61,50 @@ where
...
+            self.io.write_all(k.to_string().as_bytes()).await?;
```
Comment (vavrusa) [inline src/ascii.rs]: `k.into_bytes()` but WriteAll shouldn't require owned Vec, so `k.as_bytes()` should work without extra copy

File: src/ascii.rs [hunk 3]
```diff
@@ -60,6 +61,50 @@ where
...
+        }
+        const CRLF: [u8; 2] = [13, 10];
```
Comment (vavrusa) [inline src/ascii.rs]: `b"\r\n"`

**Review by vavrusa — CHANGES_REQUESTED** [review]

File: src/ascii.rs (hunk 2 above)
Comment (ipostelnik) [reply src/ascii.rs]: The problem is k is Display which requires conversion to string first.

**Review by vavrusa — APPROVED** [review]

**Pull request merged**